### PR TITLE
Update smite dialog

### LINKED ISSUE #2754: Update Smite dialog with the latest product specs
Change the %s to the points

https://www.figma.com/file/qQoBbyyDXkuHwoqHIOGjEu/Colony-Simplified?node-id=4445%3A2

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Update smite dialog based on new specs

**New stuff** ✨

* Added ManageReputationDialog between the colony actions dialog and the smite dialog

**Changes** 🏗

* Changed reputation amount representation from percentages to points on the smite dialog
* Updated copies of the smite dialog

FireShot Capture 450 - Colony - localhost

FireShot Capture 451 - Colony - localhost

FireShot Capture 451 - Colony - localhost

FireShot Capture 453 - Colony - localhost

Resolves #2754 and #2718

### CODE REVIEW

File: src/img/icons/emoji-shooting-star.svg
Comment (rdig) [inline src/img/icons/emoji-shooting-star.svg:1]: This SVG doesn't render properly for some reason:

Screenshot from 2021-09-30 20-39-11

Notice how it's smaller than the smite emoji. Please get a proper one from Karol (or the internet). If that doesn't work out, let me know, and I'll help

File: src/modules/dashboard/components/ManageReputationDialog/ManageReputationDialog.tsx
Comment (rdig) [inline src/modules/dashboard/components/ManageReputationDialog/ManageReputationDialog.tsx:51]: This is not something we should do right now, but at some point, I suggest we move all dialog components under something like: `~dashboard/Dialogs/...` and then we can remove the `Dialog` part from the component's name.

So this one would become: `/src/modules/dashboard/components/Dialogs/ManageReputation`

Thoughts @ArmandoGraterol @alicjakujawa @chinins ?

File: src/modules/dashboard/components/SmiteDialog/SmiteDialogForm.tsx
Comment (rdig) [inline src/modules/dashboard/components/SmiteDialog/SmiteDialogForm.tsx:332]: Please refactor this to use `FormattedMessage` and values. It will be much more readable.

**Review by rdig — APPROVED** [review]
Very nice work.

I found some things that need to be fixed / improved, but I'm pre-approving so that after you're done addressing them, you can merge.

File: src/modules/dashboard/components/ManageReputationDialog/ManageReputationDialog.tsx
Comment (ArmandoGraterol) [reply src/modules/dashboard/components/ManageReputationDialog/ManageReputationDialog.tsx:51]: That sounds like a good idea to me!

File: src/img/icons/emoji-shooting-star.svg
Comment (ArmandoGraterol) [reply src/img/icons/emoji-shooting-star.svg:1]: Interesting, I think I know what it is, it should be simple to fix

File: src/modules/dashboard/components/ManageReputationDialog/ManageReputationDialog.tsx
Comment (chinins) [reply src/modules/dashboard/components/ManageReputationDialog/ManageReputationDialog.tsx:51]: Totally agree! I can create an issue for that in the maintenance project if that's ok with everybody?

**Review by chinins — APPROVED** [review]
Very nice work!

**Pull request merged**
